src/components/InputsTable.js

```diff
@@ -0,0 +1,205 @@
+    configuredInputDefinition, setConfiguredInputDefinition,
+    inputTableSort, setInputTableSort,
+    missingRequiredInputs, missingExpectedAttributes
+  } = props
+
+  const [structBuilderVisible, setStructBuilderVisible] = useState(false)
+  const [structBuilderPath, setStructBuilderPath] = useState([])
+
+  const dataTableAttributes = _.keyBy('name', selectedDataTable.attributes)
+
+  const inputTableData = _.flow(
+    _.entries,
+    _.map(([index, row]) => {
+      const { workflow, call, variable } = parseMethodString(row.input_name)
+      return _.flow([
+        _.set('taskName', call || workflow || ''),
+        _.set('variable', variable || ''),
+        _.set('inputTypeStr', Utils.renderTypeText(row.input_type)),
+        _.set('configurationIndex', parseInt(index))
+      ])(row)
+    }),
+    _.orderBy([({ [inputTableSort.field]: field }) => _.lowerCase(field)], [inputTableSort.direction])
+  )(configuredInputDefinition)
+
+  const recordLookupWithWarnings = rowIndex => {
+    const currentInputName = _.get(`${rowIndex}.input_name`, inputTableData)
+
+    return div({ style: { display: 'flex', alignItems: 'center', width: '100%', paddingTop: '0.5rem', paddingBottom: '0.5rem' } }, [
+      RecordLookupSelect({
+        source: _.get(`${inputTableData[rowIndex].configurationIndex}.source`, configuredInputDefinition),
+        dataTableAttributes,
+        updateSource: source => {
+          setConfiguredInputDefinition(
+            _.set(`${inputTableData[rowIndex].configurationIndex}.source`, source, configuredInputDefinition))
+        }
+      }),
+      missingRequiredInputs.includes(currentInputName) && h(TooltipTrigger, { content: 'This attribute is required' }, [
+        icon('error-standard', {
+          size: 14, style: { marginLeft: '0.5rem', color: colors.warning(), cursor: 'help' }
+        })
+      ]),
+      missingExpectedAttributes.includes(currentInputName) && h(TooltipTrigger, { content: 'This attribute doesn\'t exist in data table' }, [
+        icon('error-standard', {
+          size: 14, style: { marginLeft: '0.5rem', color: colors.warning(), cursor: 'help' }
+        })
+      ])
+    ])
+  }
+
+  const parameterValueSelect = rowIndex => {
+    return ParameterValueTextInput({
+      id: `input-table-value-select-${rowIndex}`,
+      source: _.get(`${inputTableData[rowIndex].configurationIndex}.source`, configuredInputDefinition),
+      updateSource: source => {
+        setConfiguredInputDefinition(
+          _.set(`${inputTableData[rowIndex].configurationIndex}.source`, source, configuredInputDefinition))
+      }
+    })
+  }
+
+  const structBuilderLink = rowIndex => {
+    return h(StructBuilderLink, {
+      structBuilderVisible,
+      onClick: () => {
+        setStructBuilderVisible(true)
+        setStructBuilderPath([rowIndex])
+      }
+    })
+  }
+
+  return h(AutoSizer, [({ width, height }) => {
+    return h(div, {}, [
+      structBuilderVisible ? h(StructBuilderModal, {
```

Comment:
This can be updated to `structBuilderVisible && h()` so that you don't have to do the ternary condition.